src/com/palmergames/bukkit/towny/Towny.java

```diff
@@ -483,141 +488,52 @@ public void onDisable() {
 		Bukkit.getLogger().info("=============================================================");
 	}
 	
-	private void checkPlugins() {
+	private void checkSupport() {
+		final Map<String, SupportUtil.Support> results = SupportUtil.test();
 		
-		plugin.getLogger().info("Searching for third-party plugins...");
-		String ecowarn = "";
-		List<String> addons = new ArrayList<>();
-		Plugin test;
-
-	 	//Check for permission source.
-		String permissions = returnPermissionsProviders();
-		String economy = "";
-		/*
-		 * Check for economy source.
-		 */
-		if (TownySettings.isUsingEconomy()) {			
-			if (TownyEconomyHandler.setupEconomy()) {
-				economy = "  Economy: " + TownyEconomyHandler.getVersion();				
-				if (TownyEconomyHandler.isEssentials())
-					ecowarn = "Warning: EssentialsX Economy has been known to reset town and nation bank accounts on rare occasions.";
-				
-		        File f = new File(TownyUniverse.getInstance().getRootFolder(), "debtAccountsConverted.txt");                   // For a short time Towny stored debt accounts in the server's
-		        if (!f.exists())                                                                                               // economy plugin. This practice had to end, being replaced 
-		        	Bukkit.getScheduler().runTaskLaterAsynchronously(this, () -> MoneyUtil.convertLegacyDebtAccounts(), 600l); // with the debtBalance which is stored in the Town object.
-					
-			} else {
-				ecowarn = "Warning: No compatible Economy plugins found. Install Vault.jar or Reserve.jar with any of the supported eco systems. If you do not want an economy to be used, set using_economy: false in your Towny config.yml.";
+		// Compatibility report
+		if (!results.isEmpty()) {
+			plugin.getLogger().warning(Translation.of("msg_compat_found", results.size()));
```

Comment:
Should not be warning level, extension plugins are also marked as 'warnings' by this.